editors/code/src/util.ts

```diff
@@ -0,0 +1,18 @@
+let ENABLED: boolean = false;
+
```

Comment:
It looks to me that the better filename for this would be `logging.ts`. Also, I am not a fan of `utils, helper`, etc. naming.

Reply:
I want to add assert to this module as well. Use strict as assert is not really intuitive

Reply:
Yeah, mixing asserts, logs and stuff in one file is what I am not a fan of...